optuna/visualization/parallel_coordinate.py

```diff
@@ -7,21 +7,17 @@
 from optuna.visualization.utils import _check_plotly_availability
 from optuna.visualization.utils import is_available
 
-if type_checking.TYPE_CHECKING:
-    from typing import DefaultDict  # NOQA
-    from typing import List  # NOQA
-    from typing import Optional  # NOQA
+from typing import DefaultDict  # NOQA
+from typing import List  # NOQA
+from typing import Optional  # NOQA
+from optuna.study import Study  # NOQA
 
-    from optuna.study import Study  # NOQA
-
-if is_available():
-    from optuna.visualization.plotly_imports import go
+from optuna.visualization.plotly_imports import go
```

Comment:
We need to keep this checking. Otherwise, it raises an exception if the user who doesn't install `plotly` imports this module.

```suggestion
if is_available():
    from optuna.visualization.plotly_imports import go
```